plugins/modules/win_psrepository_copy.ps1

```diff
@@ -0,0 +1,234 @@
+#!powershell
+
+# Copyright: (c) 2020, Brian Scholer <@briantist>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+
+$spec = @{
+    supports_check_mode = $true
+    options = @{
+        source = @{
+            type = 'path'
+            default = '%LOCALAPPDATA%\Microsoft\Windows\PowerShell\PowerShellGet\PSRepositories.xml'
```

Comment:
Curious, does this path ever change. Just wondering why someone would want to explicitly set this? I would assume it's if you want to copy it from another user's local app data dir?

Reply:
The path where `PowerShellGet` reads it from never changes as far I can tell. The main reason to explicitly set this is if you aren't trying to use the running user (or any existing user's) file as a source (they may have changed it after all), like if you wanted to maintain a known source set in a specific location (or copy it in on execution from Ansible to a temporary location, like I did in the tests).

Reply:
Thanks for the explanation.